tracks/elm/exercises/twelve-days/mentoring.md

```diff
@@ -0,0 +1,41 @@
+---
+
+This is a good exercise to talk about naming, as it is quite difficult and most students struggle with it.
+
+This exercise introduces some List handling functions, such as `List.map`, `List.range`, `List.take` and `List.reverse`, and some string handling functions, such as `++` and `String.join`.
+
+Students should strive to communicate what the code does in as few lines as possible (ideally you should be able to understand what is it trying to do in the entry function). They show also strive to communicate how it does it in as few lines as possible.
+
+### Reasonable solutions
+
+Simply listing out the complete text of all verses is the simplest and easiest solution, and would be a reasonable initial choice in real life. Then, if changes were required to the song, it would make sense to refactor to remove the duplication, and then make the changes. However this solution doesn't really help the student learn elm, so if they have done this it makes sense to encourage them to make a further iteration.
```

Comment:
```suggestion
Only listing out the complete text of all verses is the most straight-forward solution, and is a reasonable initial choice in real life. Then, if changes are required to the song, it makes sense to refactor to remove the duplication, and then make the changes. However this solution doesn't really help the student learn elm, so if they have done this encourage them to make a further iteration.
```

Remove some of the wording so this is less windy, whilst conveying the same information, as well as simplifying the tense. I love the content :heart: